ChaosMod/Effects/db/Player/PlayerBigBang.cpp

```diff
@@ -90,11 +90,9 @@ static void OnStart()
 		Vector3 entityCoord = GET_ENTITY_COORDS(entity, false);
 		if (GET_DISTANCE_BETWEEN_COORDS(entityCoord.x, entityCoord.y, entityCoord.z, playerPos.x, playerPos.y, playerPos.z, true) <= 20)
 		{
-			//boom
-			for (int i = 0; i < 30; i++)
-			{
-				APPLY_FORCE_TO_ENTITY(entity, 1, entityCoord.x - playerPos.x, entityCoord.y - playerPos.y, entityCoord.z - playerPos.z, 0, 0, 0, 0, false, false, true, false, true);
-			}
+			
+			APPLY_FORCE_TO_ENTITY(entity, 1, (entityCoord.x - playerPos.x) * 200, (entityCoord.y - playerPos.y) * 200 , (entityCoord.z - playerPos.z) * 400, 0, 0, 0, 0, false, false, true, false, true);
```

Comment:
Thinking of it: I should have provided Functions for it. 
so „Vector3 forceVec = (entityCoord - playerPos) * 200;“ should be possible 🤔

Reply:
I'll fix this soon